build.gradle

```diff
@@ -20,7 +20,7 @@ plugins {
     id 'org.sonarqube' version '3.3'
     id 'org.springframework.boot' version '2.4.12'
     id "org.flywaydb.flyway" version "8.3.0"
-    id 'au.com.dius.pact' version '4.1.7'// do not change, otherwise serenity report fails
+    id 'au.com.dius.pact' version '4.3.6'// do not change, otherwise serenity report fails
```

Comment:
hi @ishratt - did you check if the serenity report is being generated for this upgraded version?